template/_dynamic_files.jinja2

```diff
@@ -932,6 +932,8 @@ This page discusses how to update the contents of your imports, like adding or r
 
 ## Importing a new term
 
+Note: some ontologies now use a merged-import to manage dynamic imports, for these please follow instructions in the section title "Using the Base Module approach".
+
 Importing a new term is split into to sub-phases:
```

Comment:
Should "to" be "two"?
Consider "phases" instead of "sub-phases".

Reply:
I think "to" is correct, will add system behind "merged-import" to clarify. 
Rather not change someone else's writing for the steps part >.< heh

Reply:
whoops - could you help me make the edit and make a PR please :) thanks

Reply:
Sure, @shawntanzk! I currently don't have write access to INCATools/ontology-development-kit - can give me write access?

Reply:
ah dang, I dont have permissions to give you permissions - no worries ill do it :) thanks

Reply:
> @dosumis, could you grant me write access to INCATools/ontology-development-kit ?

Done